src/shadowbox/docker/Dockerfile

```diff
@@ -38,8 +38,15 @@ WORKDIR /root/shadowbox
 
 RUN mkdir bin && curl -SsL https://github.com/Jigsaw-Code/outline-ss-server/releases/download/v${SS_VERSION}/outline-ss-server_${SS_VERSION}_linux_x86_64.tar.gz | tar xz -C bin outline-ss-server
 COPY third_party/prometheus/prometheus ./bin/
-COPY src/shadowbox/package.json .
+COPY package.json .
 COPY yarn.lock .
+
+# Copy yarn workspace dependencies so `yarn install` can reference them.
+# Note that shadowbox app is already built and only depends on outline-metrics-server types.
+COPY src/metrics_server/index.ts ./src/metrics_server/
```

Comment:
I'd like us to move to a more sane build. This manual tracking of the dependencies is problematic. Instead, you can call webpack on index.ts in the build. This way you'll pull all the dependencies automatically. Make sure you set the [target](https://webpack.js.org/configuration/target/) to `node`.

Reply:
Are you suggesting I use webpack in the shadowbox build?